annotate.el

```diff
@@ -228,30 +228,37 @@ it is called any time the buffer content is changed (so, for
 example, text is added or deleted). In particular, it will
 rearrange the overlays bounds when an annotated text is
 modified (for example a newline is inserted)."
-  (annotate-with-inhibit-modification-hooks
-   (save-excursion
-     (let* ((bol (annotate-beginning-of-line-pos))
-            (eol (annotate-end-of-line-pos))
-            (ov  (cl-remove-if-not 'annotationp
-                                   (overlays-in bol eol))))
-       (dolist (overlay ov)
-         (annotate--remove-annotation-property (overlay-start overlay)
-                                               (overlay-end   overlay))
-         ;; move the overlay if we are breaking it
-         (when (<= (overlay-start overlay)
-                   a
-                   (overlay-end overlay))
-           (move-overlay overlay (overlay-start overlay) a)
-           ;; delete overlay if there is no more annotated text
-           (when (annotate-annotated-text-empty-p overlay)
-             (delete-overlay overlay))))))))
+  (when (buffer-file-name)
```

Comment:
We should investigate why this is necessary, as this effectively breaks annotate-mode if a file without a name is being edited.

Could you tell us the error message you are getting if this is left in?